bindings/python/opendaq/generated/signal/py_mock_signal.cpp

```diff
@@ -68,14 +67,148 @@ void MockSignal::addData(const py::array_t<double>& data)
     auto valuePacket = daq::DataPacketWithDomain(domainPacket, signal.getDescriptor(), size);
 
     auto dataPtr = static_cast<daq::SampleTypeToType<daq::SampleType::Float64>::Type*>(valuePacket.getRawData());
-    for (auto i = 0; i < size; i++)
+    for (auto i = 0; i < size; ++i)
         dataPtr[i] = data.at(i);
 
     signal.sendPacket(valuePacket);
     domainSignal.sendPacket(domainPacket);
     offset += size;
 }
 
+void MockSignal::addObjects(const py::object& objects, bool updateDescriptor)
+{
+    daq::DataDescriptorPtr descriptor;
+    size_t size = 1;
+    if (py::isinstance<py::dict>(objects) || py::isinstance<py::list>(objects))
```

Comment:
Is outer `if` necessary?

Reply:
Yep. Just to filter non dict or list values passed